modules/oe_webtools_media/src/Plugin/Field/FieldFormatter/WebtoolsSnippetFormatter.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_media\Plugin\Field\FieldFormatter;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\Core\Field\FormatterBase;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\json_field\JsonMarkup;
+
+/**
+ * Plugin implementation of the 'webtools_snippet' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "webtools_snippet",
+ *   label = @Translation("Webtools snippet"),
+ *   field_types = {
+ *     "json",
+ *   }
+ * )
+ */
+class WebtoolsSnippetFormatter extends FormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $element = [];
+
+    foreach ($items as $delta => $item) {
+      $element[$delta] = [
+        '#type' => 'html_tag',
+        '#tag' => 'script',
+        '#value' => JsonMarkup::create(Json::encode(Json::decode($item->get('value')->getValue()))),
```

Comment:
Add a comment here:

```
// We need to properly escape the json content before outputting in the page. We cannot use twig escaping
// mechanism as it will use html entities. Json::encode takes care of it.
```